config/config.go

```diff
@@ -128,6 +128,7 @@ type Remote struct {
 	Insecure                bool                         `json:"insecure"`
 	ConnectionCheckInterval time.Duration                `json:"connection_check_interval,omitempty"`
 	ReconnectInterval       time.Duration                `json:"reconnect_interval,omitempty"`
+	RefreshInterval         time.Duration                `json:"refresh_interval,omitempty"`
```

Comment:
while I agree with this change, you will also have to make changes to proto_conversion.go plus the api repo (since the remote config proto will have changed) and then update the app repo after that. so we can leave this change out of this PR and make a ticket or make the rest of the changes too

Reply:
👍 makes sense, ill remove it